test-poc/framework/pom.xml

```diff
@@ -66,6 +66,41 @@
             <version>${selenium.version}</version>
             <type>pom</type>
         </dependency>
+
+        <!-- Junit 4 is needed by Testcontainers. See https://github.com/testcontainers/testcontainers-java/issues/970 -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
From https://github.com/testcontainers/testcontainers-java/issues/970 it seems there's a workaround of just adding some dummy classes: https://github.com/testcontainers/testcontainers-java/issues/970#issuecomment-625044008

That would be better than adding junit4 as a dependency

Reply:
Done